!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  blob_filter_by_area: true
  blob_filter_by_circularity: true
  blob_filter_by_color: true
  blob_filter_by_convexity: true
  blob_filter_by_inertia: true
  blob_filter_by_orientation: true
  blob_max_angle: 0.5
  blob_max_area: 5000
  blob_max_circularity: 0.8
  blob_max_convexity: 1.0
  blob_max_inertia_ratio: 0.5
  blob_min_angle: 0.0
  blob_min_area: 400
  blob_min_circularity: 0.5
  blob_min_convexity: 0.75
  blob_min_dist_between: 10.0
  blob_min_inertia_ratio: 0.1
  blob_min_repeatability: 1
  blob_threshold_step: 500
  blob_threshold_width: 100
  blur_empty_areas: true
  dilate_iterations: 0
  erode_iterations: 4
  gaussianblur_size: 0
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        dictitems:
          Params: !!python/object/new:dynamic_reconfigure.encoding.Config
            dictitems:
              blob_filter_by_area: true
              blob_filter_by_circularity: true
              blob_filter_by_color: true
              blob_filter_by_convexity: true
              blob_filter_by_inertia: true
              blob_filter_by_orientation: true
              blob_max_angle: 0.5
              blob_max_area: 5000
              blob_max_circularity: 0.8
              blob_max_convexity: 1.0
              blob_max_inertia_ratio: 0.5
              blob_min_angle: 0.0
              blob_min_area: 400
              blob_min_circularity: 0.5
              blob_min_convexity: 0.75
              blob_min_dist_between: 10.0
              blob_min_inertia_ratio: 0.1
              blob_min_repeatability: 1
              blob_threshold_step: 500
              blob_threshold_width: 100
              blur_empty_areas: true
              dilate_iterations: 0
              erode_iterations: 4
              gaussianblur_size: 0
              groups: !!python/object/new:dynamic_reconfigure.encoding.Config
                state: []
              id: 1
              max_depth: 13000
              medianblur_size: 0
              min_depth: 300
              name: Params
              parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
                state: []
              parent: 0
              state: true
              type: ''
              use_threshold_width: false
            state: []
        state: []
      id: 0
      name: Default
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      state: true
      type: ''
    state: []
  max_depth: 13000
  medianblur_size: 0
  min_depth: 300
  use_threshold_width: false
state: []
